hack/validate-yaml

```diff
@@ -10,14 +10,36 @@ function main(){
 
     FAILED_CFGS_YML=()
     FAILED_CFGS_KB=()
+    FAILED_CHECKS_TYPE=()
 
     # Loop through all benchmarks
     for cfg in ${CFGS}; do
     set -x
         # yammlint is configured with ../.yamllint.yaml, to catch any liniting errors for a given benchmark. If result isn't empty, FAILED_CFGS_YML is appended with the failing benchmark.
         if [ -n "$(yamllint -s package/cfg/$cfg)" ]; then
-            FAILED_CFGS_YML+=("package/cfg/$cfg")
+            FAILED_CFGS_YML+=("$cfg")
         fi
+        # verify each check content to confirm type Automated matches scored to true and type Manual matches scored to false.
```

Comment:
Good point, the `type` is independent of the `scored` variable in kube-bench. All that matters for now, is to make sure that:

-  Title `Automated` is mapped to `scored: true` (this will generate a PASS if it complies, and a FAIL if it's not compliant). 
-  Title `Manual` is mapped to `scored: false` (this will generate a PASS if it complies, and a WARN if it's not compliant).

The type skip and manual in kube-bench have been added in kube-bench to give more flexibility to end-user, and don't exist in CIS Workbench.